source/developers-guide/shopware-5-rest-api/api-resource-translation/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Shopware 5 Rest API - Translations End-Point
+github_link: developers-guide/shopware-5-rest-api/api-resource-translation/index.md
+indexed: true
+---
+
+## Introduction
+
+In this part of the documentation you can learn more about the API's translation resource. With this resource, it is possible to 
+receive, delete and update any translation in your shops. Also we will have a look at the provided data.
+
+## General Information
+You may find the related resource under
+**engine\Shopware\Controllers\Api\Shops.php**.
+
+This resource supports the following operations:
+
+|  Access URL                 | GET                | GET (List)      | PUT             | PUT (Stack)      | POST             | DELETE          | DELETE (Stack)  |
+|-----------------------------|--------------------|-----------------|-----------------|------------------|------------------|-----------------|-----------------|
+| /api/translations	          | ![No](./img/no.png)     | ![Yes](./img/yes.png) | ![Yes](./img/yes.png) | ![Yes](./img/yes.png)  | ![Yes](./img/yes.png)  | ![Yes](./img/yes.png) | ![Yes](./img/yes.png) |
+
+If you want to access this end-point, simply append your shop-URL with
+
+* **http://my-shop-url/api/translations**
+
+## GET (List)
+
+You can receive data of translations by providing the specific id
+
+* **http://my-shop-url/api/translations/id**
+
+Simply replace the 'id' with the specific identifier
+
+### Return Value
+
+| Field               | Type                  | Original object                                 |
+|---------------------|-----------------------|-------------------------------------------------|
+| type 	         	  | string				  |                                                 |
+| data		      	  | array				  | 		                                        |
+| key				  | integer 			  | 												|
+| localeId	      	  | integer (foreign key) | **[Locale](./models/locale)**                    |
+| locale			  | object				  | **[Locale](./models/locale)**					|
+
+*Since this returns a list, the following fields will be added to the array:*
+
+| Field               | Type                  | Comment			                                |
+|---------------------|-----------------------|-------------------------------------------------|
+| total				  | integer				  | The total amount of cache resources             |
+| success		      | boolean				  | Indicates if the call was stressful or not.		|
+
+## POST
+To post a translation, you need to identify it by the following parameters
```

Comment:
a POST creates a new translation, so it's not clear why you need to identify it. Clarify this please